doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.rst

```diff
@@ -490,18 +433,30 @@ Now, we add an example stage to the node. The first line sets ``current_state_pt
       current_state_ptr = stage_state_current.get();
       task.add(std::move(stage_state_current));
 
-In order to plan any robot motions, we need to specify a solver. MoveIt Task Constructor has three options for solvers:
+Solvers are used to define the type of robot motion. MoveIt Task Constructor has three options for solvers:
 
- * ``PipelinePlanner`` uses MoveIt's planning pipeline, which typically defaults to OMPL.
- * ``CartesianPath`` is used to move the end effector in a straight line in Cartesian space.
- * ``JointInterpolation`` is a simple planner that interpolates between the start and goal joint states. It is typically used for simple motions as it computes quickly but doesn't support complex motions.
 
-We also set some properties specific for to the Cartesian planner.
+  **PipelinePlanner** uses MoveIt's planning pipeline, which typically defaults to `OMPL <https://github.com/ompl/ompl>`_.
 
-.. code-block:: c++
+  .. code:: c++
 
-      auto sampling_planner = std::make_shared<mtc::solvers::PipelinePlanner>(node_);
-      auto interpolation_planner = std::make_shared<mtc::solvers::JointInterpolationPlanner>();
+        auto sampling_planner = std::make_shared<mtc::solvers::PipelinePlanner>(node_);
+
+  **JointInterpolation** is a simple planner that interpolates between the start and goal joint states. It is typically used for simple motions as it computes quickly but doesn't support complex motions.
+
+  .. code:: c++
+
+        auto interpolation_planner = std::make_shared<mtc::solvers::JointInterpolationPlanner>();
+
+  **CartesianPath** is used to move the end effector in a straight line in Cartesian space.
+
+  .. code:: c++
+
+        auto cartesian_planner = std::make_shared<mtc::solvers::CartesianPath>();
+
+Feel free to try out the different solvers and see how the robot motion changes. For this tutorial we will use the Cartesian planner, which requires the following properties to be set:
```

Comment:
We use all three solvers in this tutorial - in the minimal example, we use the [joint interpolation planner ](https://github.com/ros-planning/moveit2_tutorials/blob/628bcfaceed8b992d72b737066d4ac635b055388/doc/tutorials/pick_and_place_with_moveit_task_constructor/src/minimal.cpp#L133) in our open hand stage, and as we add the pick and place stages, the Cartesian planner and pipeline planners are also used.